### DISCUSSION TITLE
New engine info?

### DISCUSSION THREAD

**Discussion by asmaloney** [The-Wineskin-Project/WineskinServer#343] [Q&A] [answered]
I see a new engine has been added - `WS11WineCX64Bit22.0.1-1`.

Where do we find changelogs for these engine releases?

Are there any update instructions anywhere for when we change engines? I don't know exactly what is affected (as I still don't understand what's in the engine, what's in the wrapper, and what's in winetricks), so I don't know if we need to do anything else e.g. update & rerun winetricks as well?

Thanks Dean!

**Comment by Gcenx** [comment, marked as answer]
> I see a new engine has been added - `WS11WineCX64Bit22.0.1-1`.
> 
> Where do we find changelogs for these engine releases?

Honestly I’ve never done change logs for Engines. CX22.0.1-1 has some cherry picked commits from upstream along with dropping gstreamer due to it being a PITA when packaging.

> Are there any update instructions anywhere for when we change engines?

That’s really upto you when or if you decide to upgrade the Engine. If everything is working within a port might be better to simply leave it alone.

> I still don't understand what's in the engine

Engines are just nicely packaged wine builds.

> what's in the wrapper

The master wrapper is the libraries/utility’s to help create your own custom “port”

> and what's in winetricks

winetricks is simply a wrapper script that’s used to install some windows dependencies and commonly includes workarounds to get them to install/setup correctly for wine.
